libpod/define/annotations.go

```diff
@@ -175,3 +181,118 @@ func IsReservedAnnotation(value string) bool {
 		return false
 	}
 }
+
+// RegexErrorMsg returns a string explanation of a regex validation failure.
+func RegexErrorMsg(msg string, fmt string, examples ...string) string {
+	if len(examples) == 0 {
+		return msg + " (regex used for validation is '" + fmt + "')"
+	}
+	msg += " (e.g. "
+	for i := range examples {
+		if i > 0 {
+			msg += " or "
+		}
+		msg += "'" + examples[i] + "', "
+	}
+	msg += "regex used for validation is '" + fmt + "')"
+	return msg
+}
+
+const dns1123LabelFmt string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
+const dns1123SubdomainFmt string = dns1123LabelFmt + "(\\." + dns1123LabelFmt + ")*"
+const dns1123SubdomainErrorMsg string = "a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character"
```

Comment:
This should mention that annotations must be a RFC1123 subdomain - maybe "annotations must be formatted as a valid lowercase RFC1123 subdomain of lower case..."

Reply:
This message is copied from Kubenetes code as-is. Still change to suggested one?

Reply:
I see no reason to tie ourselves to Kubernetes in every aspect. The error message could be better and should be changed.